package.json

```diff
@@ -12,13 +12,14 @@
     "devnet": "ENABLE_MAINNET_FORKING=true NO_RESET=true npx hardhat node",
     "console:rinkeby": "npx hardhat console --network rinkeby",
     "console:ropsten": "npx hardhat console --network ropsten",
-    "clean": "rm -rf build && rm -rf out",
-    "test": "forge test",
+    "clean": "forge clean",
+    "test": "forge test --no-match-contract IntegrationTest",
+    "test:integration": "forge test --fork-url https://eth-mainnet.alchemyapi.io/v2/$MAINNET_ALCHEMY_API_KEY --fork-block-number 14207980 --match-contract UniswapV3OracleIntegrationTest",
```

Comment:
maybe don't hardcode the block number in the package.json, it's faster to do so but it's going to be a pain to debug later if we have an issue because a contract is deployed after this block & we try to use it

also --match-contract is too specific

Reply:
Yeah good point, I shouldn't have hard coded - will move to the `.env.example`. 

In terms of `--match-contract` I need a way to distinguish the integration tests from the unit tests. It matches based on regex, so best way is likely to use `--match-contract` but make it more general than what I have now. Something like:

`--match-contract IntegrateTest`